source/ore/api.rst

```diff
@@ -69,3 +69,27 @@ Below is a list of the following routes that are currently available within the
     routes/list-users
     routes/user
     routes/download
+
+
+Deprecated Routes
+~~~~~~~~~~~~~~~~~~
+
+The following routes are deprecated and should not be used the replacements are likley to differ from the old one so consult the docs before migrating.
```

Comment:
`used. The replacements`

Reply:
Also wrap this line to 120 chars as mentioned https://docs.spongepowered.org/stable/en/contributing/spongedocs.html#style-guide